src/controllers/UserController.js

```diff
@@ -388,6 +388,46 @@ class UserController {
       return HelperMethods.serverError(res);
     }
   }
+
+  /**
+   * Update rememberDetails column of a user
+   * Route: POST: api/v1/
+   * @param {object} req - HTTP Request object
+   * @param {object} res - HTTP Response object
+   * @return {res} res - HTTP Response object
+   * @memberof UserController
+   */
+  static async notify(req, res) {
+    const { id } = req.params;
+    const notify = `
+    <!DOCTYPE html>
+<html lang="en">
+<head>
+ <meta charset="UTF-8">
+ <meta http-equiv="X-UA-Compatible" content="ie=edge">
+ <title>Document</title>
+ <script src="/socket.io/socket.io.js"></script>
+</head>
+<body>
+  <script>
+    const messageDisplay = (message) => {
+    const div = document.getElementById('message-container');
+    let paragraph = document.createElement('p');
+    paragraph.appendChild(document.createTextNode(message));
+    div.appendChild(paragraph);
+    };
+    const socket = io();
+       socket.on('${id}', (data) => {
```

Comment:
This is super brilliant! Thumbs up ! You might consider removing the braces around id since it's already a string.

Reply:
there are braces from the emit function too... so removing it here would not work

Reply:
Oh. I was thinking socket.on(id, (data) => ........ would do the job